server/core/internal/config.h

```diff
@@ -28,7 +28,8 @@ MXS_BEGIN_DECLS
 #define DEFAULT_NTHREADS            1    /**< Default number of polling threads */
 #define DEFAULT_QUERY_RETRIES       0    /**< Number of retries for interrupted queries */
 #define DEFAULT_QUERY_RETRY_TIMEOUT 5    /**< Timeout for query retries */
-
+#define DEFAULT_WRITEQ_HIGH_WATER   8096 /**< Defalut high water mark of dcb write queue */
```

Comment:
Disable throttling by default to keep current behavior. The value `0` can be used to tell that the throttling is disabled.